ui/components/component-library/avatar-account/__snapshots__/avatar-account.test.js.snap

```diff
@@ -0,0 +1,50 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`AvatarAccount should render correctly 1`] = `
+<div>
+  <div
+    class="box mm-avatar-base mm-avatar-base--size-md mm-avatar-account box--flex-direction-row box--color-text-default box--background-color-background-alternative box--border-color-border-default box--border-style-solid box--border-width-1"
+    data-testid="avatar-account"
+  >
+    <div
+      class="mm-avatar-account__jazzicon"
+    >
+      <div
+        style="border-radius: 50px; overflow: hidden; padding: 0px; margin: 0px; width: 32px; height: 32px; display: inline-block; background: rgb(249, 108, 1);"
+      >
```

Comment:
I wonder if there is anyway we could reduce this nesting? Is it possible to add the `class="mm-avatar-account__jazzicon"` to the div with all the inline styles? it may not be..

Reply:
Ah ok you are adding the class directly to the jazzicon. That library has a lot of nesting divs wow haha